src/Views/listings/view.phtml

```diff
@@ -14,22 +14,22 @@ $title = 'listing';
     </div>
     <div class="view-listing-header listing-grid-design">
         <div>
-            <h1> <?=htmlspecialchars($listing->getListingName()) ?></h1>
+            <h1><?=htmlspecialchars($listing->getListingName()) ?></h1>
```

Comment:
Sorry I didn't say earlier, but for consistency all the php tags should have a space at the start and end